modules/RTC/RTC.js

```diff
@@ -83,6 +83,13 @@ export default class RTC extends Listenable {
         // this happen
         this.dataChannelsOpen = false;
 
+        // Defines the last N endpoints list. It can be null or an array once
+        // initialised with a datachannel last N event.
+        this._lastNEndpoints = null;
```

Comment:
It helps if the type is declared like this:
`/*`
  `* @type {Array<string>|null}`
  `*/`
This way IDE and some other tools are able to detect the type.